tests/integration/policies/authn-gcp.yml

```diff
@@ -0,0 +1,16 @@
+- !policy
+  id: authn-gcp
+  body:
+    - !webservice
+
+    - !group clients
+
+    - !permit
+      role: !group clients
+      privilege: [ read, authenticate ]
+      resource: !webservice
+
+    - !permit
+      role: !user /test-authenticator-valid-user
+      privilege: [ update ]
+      resource: !webservice
```

Comment:
Newline

Reply:
It's on purpose. Only Python requires an empty newline in the end